repository_service_tuf_api/metadata.py

```diff
@@ -84,8 +84,13 @@ def post_metadata(payload: MetadataPostPayload) -> MetadataPostResponse:
     return MetadataPostResponse(data=data, message=message)
 
 
+class RolesData(BaseModel):
+    root: TUFMetadata
+    trusted_root: Union[TUFMetadata, Dict]  # Will be {} if root is bootstrap
```

Comment:
Isn't it better to make it `Optional[TUFMetadata]`

Reply:
I did it like that as you suggested that we should always return `trusted_root` in the response.
I am okay to change that.